tests/console/btrfs_check.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: btrfs_check
+# Summary: Test that btrfs is setup properly
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    my $kiwi_volumes = qq(
```

Comment:
this will a bit more complicated. Given the fact that we have 3 archs (no ppc) the layout will differ. Especially when we are talking about boot subvolumes.

Reply:
Any clues for checking all the possibilities?